content/en/blog/falcosidekick-openfaas.md

```diff
@@ -0,0 +1,591 @@
+---
+title: Falcosidekick + OpenFaas = a Kubernetes Response Engine, Part 2
+date: 2021-04-11
+author: Batuhan Apaydın
+slug: falcosidekick-openfaas
+---
+
+We recently talked about a concept called _"Kubernetes Response Engine"_, and we achieved this by
+using `Falco` `Falcosidekick` + `Kubeless`. But as you might guess, _Falcosidekick_ project is evolving day by day,
+which means new output types is adopting to the project. With the
+release [`2.22.0`](https://github.com/falcosecurity/falcosidekick/releases/tag/2.22.0), we are proud of being
+support [`OpenFaaS`](https://www.openfaas.com) as a new output type of _Falcosidekick_, which means now, we can achieve
+the same concept called _"Kubernetes Response Engine"_ but this time by using _"OpenFaaS"_ instead of _"Kubeless"_.
```

Comment:
```suggestion
release [`2.22.0`](https://github.com/falcosecurity/falcosidekick/releases/tag/2.22.0), we are proud to
support [`OpenFaaS`](https://www.openfaas.com) as a new output for _Falcosidekick_. We can now achieve
the same concept called _"Kubernetes Response Engine"_ but this time by using _"OpenFaaS"_ instead of _"Kubeless"_.
```

Reply:
done